src/tree-sitter/csharpQueries.ts

```diff
@@ -0,0 +1,52 @@
+import { TreeSitterQueries } from './types.js'
+
+export const csharpQueries: TreeSitterQueries = {
+  defineParameterTypeQueries: [
```

Comment:
The purpose of this tree-sitter query is to capture [Cucumber Expression parameter types](https://github.com/cucumber/cucumber-expressions#parameter-types).

The Cucumber Expressions library has recently been [ported to .NET](https://github.com/cucumber/cucumber-expressions/pull/45) and released on [nuget](https://www.nuget.org/packages/Cucumber.CucumberExpressions/), but it hasn't yet been integrated into SpecFlow.

While the current `StepArgumentTransformation` offers similar functionality, it's not compatible with Cucumber Expressions. The only reason to parse parameter types is to be able to compile Cucumber Expressions found in the step definitions.

Until SpecFlow supports CucumberExpressions, I think this array should be empty.

/cc @gasparnagy @SabotageAndi

Reply:
I've taken this out for now